RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py

```diff
@@ -60,5 +75,7 @@
 phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCEEConfig , fCPerMIP = fCPerMIP_v10 ) 
 phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCHEFConfig , fCPerMIP = fCPerMIP_v10 )
 
+from Configuration.Eras.Modifier_phase2_hfnose_cff import phase2_hfnose
+phase2_hfnose.toModify( HGCalUncalibRecHit.HGCHFNoseConfig , isSiFE = True )
```

Comment:
shouldn't this be `True` all the time? Why does the value of `HGCalUncalibRecHit.HGCHFNoseConfig.isSiFE` matter if `phase2_hfnose` is not enabled?

Reply:
I can make this in my next PR where I make changes to the code which enables RecHits for HFNose